### DISCUSSION TITLE
BinaryKits.Zpl.Viewer Linux with Avalonia

### DISCUSSION THREAD

**Discussion by ricardopino** [BinaryKits/ZPLUtility#201] [Q&A]
Hello, does anyone know if it is possible to compile for Linux? I made an application for windows/linux using avalonia but compiling for linux fails:"PlatformNotSupportedException: System.Drawing.Common is not compatible with non-Windows platforms. See https://aka.ms/systemdrawingnonwindows for more information."
Sorry for my english.
Ricardo Pino.

**Comment by primo-ppcg** [comment]
System.Drawing is needed only for drawing the barcode elements with BarcodeLib. This dependency [will be dropped](https://github.com/BinaryKits/BinaryKits.Zpl/pull/171) in the near future.

**Reply by ricardopino** [reply]
Oka, gracias! Waiting for news.
rp
